modules/microshift-config-parameters-table.adoc

```diff
@@ -0,0 +1,128 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-using-config-yaml.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="microshift-config-parameters-table_{context}"]
+= Parameters and values for the {microshift-short} config.yaml file
+
+The following table explains {microshift-short} configuration YAML parameters and valid values for each:
+
+.{microshift-short} `config.yaml` parameters
+[cols="3","20%,10%,70%",options="header"]
+|===
+|Field|Type|Description
+
+|`advertiseAddress`
+|`array`
```

Comment:
I'm not sure why, but I'm troubled by the use of  `array` here, and other instances below.

Reply:
Standard for docs. Is it the backticks that trouble you, or would you prefer another word here?